src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -28,8 +29,37 @@
                       SerializingFunction)
 from ._utils import _timeout_to_deadline
 
+_TIMEOUT_WAIT_FOR_CLOSE_ONGOING_CALLS_SEC = 0.1
 _IMMUTABLE_EMPTY_TUPLE = tuple()
 
+_LOGGER = logging.getLogger(__name__)
+
+
+class _OngoingCalls:
+    """Internal class used for have visibility of the ongoing calls."""
+
+    _calls: Sequence[_base_call.RpcContext]
```

Comment:
Since this as an internal member of an internal class, we should probably be more specific and use `List` rather than `Sequence`.

Reply:
Just FYI since the introduction of the `WeakSet` I've used the `AbstractSet`